VEX/pub/libvex_ir.h

```diff
@@ -2410,7 +2410,11 @@ typedef
          back end, just as LL and SC themselves are. */
       Imbe_CancelReservation,
       /* Needed only on amd64.  It drains the iMC cache. */
-      Imbe_Drain
+      Imbe_Drain,
+      /* Might be interesting for some kinds of tools */
+      Imbe_SFence,
+      Imbe_MFence,
```

Comment:
Do we still need *Imbe_Fence*? I guess it should be equivalent to *Imbe_MFence*.